battleship.py

```diff
@@ -25,9 +25,20 @@
 Returns: None
 '''
 def makeModel(data):
+    data["rows"]=10
+    data["cols"]=10
+    data["boardsize"]=500
+    data["cellsize"]=50

```

Comment:
Calculate cell size using board size and number of rows/cols. Don't write the value directly.

Reply:
Modified Make Model Function